src/test/compile-fail/issue28498-reject-passed-to-fn.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
```

Comment:
I don't understand your question.

This, like the other two tests comparing ugeh and non-ugeh variants, is deliberately in cfail; with this PR, it is rejected by the compiler (as indicated by the errors).  Adding the escape-hatch attribute would allow it to pass the compiler, but the comments in the test explain why adding it in this case would be unsound.